src/Fabulous.XamarinForms/ViewUpdaters.fs

```diff
@@ -4,7 +4,7 @@ open Fabulous
 open Xamarin.Forms
 
 module ViewUpdaters =
-    let updateSliderMinMax (newValueOpt: (float * float) voption, node: IViewNode) =
+    let updateSliderMinMax (newValueOpt: (float * float) voption) (node: IViewNode) =
```

Comment:
I think we can probably use `struct (float * float)` here. With F# 6 there are a lot of places where you don't need to specify `struct` directly. F# will infer the intent. like so

```fsharp
let f  struct (a, b)  = a + b // <---struct tuple
f (1, 2) // inferred struct here
```